botocore/configloader.py

```diff
@@ -142,12 +143,13 @@ def raw_config_parse(config_filename, parse_subsections=True):
         path = os.path.expandvars(path)
         path = os.path.expanduser(path)
         if not os.path.isfile(path):
-            raise botocore.exceptions.ConfigNotFound(path=path)
-        cp = configparser.RawConfigParser()
+            raise botocore.exceptions.ConfigNotFound(path=_unicode_path(path))
+        cp = six.moves.configparser.RawConfigParser()
```

Comment:
I *think* this is going to fail depending on what the current state of ``sys.modules`` is since it's relying on something having munged ``sys.modules["six"]`` to add the moves attribute. You likely want to do ``from botocore.compat.six import moves as six_moves`` and then ``six_moves.configparser.RawConfigParser()``.